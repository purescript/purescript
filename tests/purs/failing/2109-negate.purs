-- @shouldFailWith UnknownName
module Main where

x = -5
