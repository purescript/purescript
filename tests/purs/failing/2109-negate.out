Error found:
in module [33mMain[0m
at tests/purs/failing/2109-negate.purs:4:5 - 4:7 (line 4, column 5 - line 4, column 7)

  Unknown value [33mnegate[0m. You're probably using numeric negation (the unary [33m-[0m operator), which the compiler replaces with calls to the [33mnegate[0m function. Please import [33mnegate[0m from module [33mPrelude[0m


See https://github.com/purescript/documentation/blob/master/errors/UnknownName.md for more information,
or to contribute content related to this error.

